packages/migrate/src/__tests__/DbPush.test.ts

```diff
@@ -339,9 +349,31 @@ describe('postgresql', () => {
     expect(removeRocketEmoji(ctx.mocked['console.info'].mock.calls.join('\n'))).toMatchInlineSnapshot(`
       Environment variables loaded from prisma/.env
       Prisma schema loaded from prisma/schema.prisma
-      Datasource "my_db": PostgreSQL database "tests-migrate", schema "public" at "localhost:5432"
+      Datasource "my_db": PostgreSQL database "tests-migrate-db-push", schema "public" at "localhost:5432"
+
+      The PostgreSQL database "tests-migrate-db-push" schema "public" at "localhost:5432" was successfully reset.
+
+      Your database is now in sync with your Prisma schema. Done in XXXms
+    `)
+    expect(ctx.mocked['console.error'].mock.calls.join('\n')).toMatchInlineSnapshot(``)
+  })
+
+  it('should work if url is prisma:// and directUrl defined', async () => {
+    ctx.fixture('schema-only-data-proxy')
+
+    prompt.inject(['n'])
```

Comment:
What's this "No" manual input for?

Reply:
To avoid actually doing a reset. The test still shows that `directUrl` is used, if it wasn't it would have failed.